chainlink/__init__.py

```diff
@@ -108,11 +108,17 @@ def _pull_image(client, image, status):
         result = {
             "data": self.client.api.inspect_container(container.id)["State"],
             "killed": killed,
-            "logs": {
+        }
+        if stage.get("logs", True):
+            result["logs"] = {
                 "stdout": container.logs(stderr=False, timestamps=True),
                 "stderr": container.logs(stdout=False, timestamps=True),
-            },
-        }
+            }
+        else:
+            result["logs"] = {
+                "stdout": "logs have been silenced for this stage",
```

Comment:
If the logs have been silenced, shouldn't the `logs` key just return a `None` value for `stdout` and `stderr`?

Reply:
Hmm, I thought the API strictly requires a string, but I think `None` would work too. Tested with unit tests on Broadway API.